azkaban-common/src/main/java/azkaban/project/JdbcProjectImpl.java

```diff
@@ -157,20 +157,7 @@ public synchronized Project createNewProject(final String name, final String des
       final User creator)
       throws ProjectManagerException {
     final ProjectResultHandler handler = new ProjectResultHandler();
-
-    // Check if the same project name exists.
-    try {
```

Comment:
I don't believe so. Even so, code which is almost the exact same exists in JdbcProjectImpl.fetchProjectByName. If it is to prevent race conditions due to a read lock or something, I think that would stop it there.